### ISSUE TITLE
buttons for website's Chilling Frost and Stealth skills should be unclickable/disabled when there is no point casting them

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica#12362]
_I'm marking this as "on hold: needs design" for Habitica's staff (especially @Tressley) to comment about whether design work is is needed for this and/or to give specifications for how the skill buttons should look or behave._

As described in the API issue https://github.com/HabitRPG/habitica/issues/12361, the Mage skill Chilling Frost needs to be cast only once each day, and the Rogue skill Stealth doesn't need to be cast again after it has already protected all incomplete Dailies. In the pull request https://github.com/HabitRPG/habitica/pull/12255 the tooltips for those skills were modified to indicate when the skills no longer need to be cast. In addition to that, **the skills' buttons should not be clickable and/or should look greyed-out or otherwise disabled when there is no longer any need to cast the skills.**

**Comment by veeeeeee** [comment]
Recommend using the same string as #12361 when this is implemented: "You've cast this skill already. It won't have any additional effect." Ideally, replace the tooltip when hovering over the skill set with this new message. 

This could also include a disabled state for when the user does not have sufficient MP to cast the skill.

**Comment by paglias** [comment]
https://github.com/HabitRPG/habitica/pull/12404 disables the buttons and uses the correct text when the spell should not be used anymore, so what's missing in this PR is just the disabled state for the button

**Comment by Tressley** [comment]
For the disabled state we just need to set `opacity: 0.5` and remove the cursor and hover state.
